src/header/value.rs

```diff
@@ -289,10 +289,33 @@ impl AsRef<[u8]> for HeaderValue {
 
 impl fmt::Debug for HeaderValue {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        f.debug_struct("HeaderValue")
-            .field("value", &EscapeBytes(self.as_ref()))
-            .field("is_sensitive", &self.is_sensitive)
-            .finish()
+        if self.is_sensitive {
+            f.write_str("Sensitive")
```

Comment:
Ah, this PR removed outputting a struct at all. So, normal output would look like this:

```
{"transfer-encoding": "chunked"}
```

And sensitive looks like this:

```
{"authorization": Sensitive}
```